features/README.md

```diff
@@ -483,7 +483,7 @@ foreach ($schema->keyspaces() as $keyspace) {
 
 ### Data Types
 
-The PHP driver for Apache Cassandra supports [a variety of datatypes](/features/core/datatypes/).
+The PHP driver for Apache Cassandra supports [a variety of datatypes](/features/datatypes/).
```

Comment:
This should be `/features/datatypes.feature` in order to work in GitHub. The rewrite rule can be ignored as that was addressed in #114.